src/MusicStore/Get-Crossgen.ps1

```diff
@@ -12,9 +12,11 @@ Write-Host -ForegroundColor Green "Installing crossgen.exe to $path"
 
 if (-not (Test-Path $path))
 {
-    New-Item -Path $path -ItemType Directory -Force
+    New-Item -Path $path -ItemType Directory -Force | Out-Null
 }
 
+$path = Get-Item $path
+
 function Get-CoreCLRVersion()
```

Comment:
(minor) IMO, it would be better to change all these "CoreCLR" references to be "NETCoreApp".

`Get-CoreCLRVersion()`
`$coreclrpackage`
`$coreclrversion`